src/topk/mod.rs

```diff
@@ -198,13 +218,28 @@ impl<T: Eq + Hash> FilteredSpaceSaving<T> {
         self.into_sorted_vec().into_iter()
     }
 
+    /// Returns a sorted vector of all items (does not consume the FSS)
+    ///
+    /// Computes in **O(k*log(k))** time.
+    pub fn sorted_items_vec(&self) -> Vec<(&T, &ElementCounter)> {
```

Comment:
Maybe leave this to users to implement? It's a bit confusing whether this returns a `Vec` of references or copies